mod/schema/UnitSet.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Slightly different from UnitCardSet.json
Used for base level schema validation of [...]Units.json files